src/Symfony/Component/Mailer/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+4.4.0
+-----
+
+ * [BC BREAK] Transports depend on `Symfony\Contracts\EventDispatcher\EventDispatcherInterface`
```

Comment:
that's not actually a BC break if the component interface extends the contracts one.

Reply:
Interface from component has methods like `addListener`, `addSubscriber` but interface from contracts has only `dispatch` method. So if some of transports call `addListener` - now it could get fatal error.